### DISCUSSION TITLE
Customization of the header

### DISCUSSION THREAD

**Discussion by uPagge** [squidfunk/mkdocs-material#5194] [Q&A] [answered]
Can I somehow add a link directly to the title? Where is the logo search bar and so on? For example, somehow add a second drop-down list.

My case. I have a library that is written for two Spring and Quarkus platforms. I also want to make multilingual documentation. And it turns out I need a second switch.

**Comment by squidfunk** [comment, marked as answer]
[Theme extension](https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme) is the way to go! I, personally, can't help you with your specific changes, but maybe other users can share their code examples here and how they extended Material for MkDocs 😊 You likely need to override the [`header.html`](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/header.html) partial.

Regarding multi-language usage, please refer to #2346.
